python/servo/package_commands.py

```diff
@@ -133,6 +137,25 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
                 checked.update(checking)
                 need_checked.difference_update(checked)
 
+            print("Adding version to Credits.rtf")
+            version_command = [binary_path, '--version']
+            p = subprocess.Popen(version_command,
+                                 stdout=subprocess.PIPE,
+                                 stderr=subprocess.PIPE,
+                                 universal_newlines=True)
+            version, stderr = p.communicate()
+            if p.returncode != 0:
+                print("Error occurred when getting Servo version: " + stderr)
+                return 1
+            version = "Nightly version: " + version
+
+            template_path = os.path.join(dir_to_resources, 'Credits.rtf.mako')
```

Comment:
Not 100% sure but it looks like this gets left in the final `dmg` - we should remove it.